app/views/competitions/shark_tank.html.erb

```diff
@@ -5,10 +5,14 @@
         <%= image_tag('shark-tank-logo.png', alt: 'Shark Tank 2020 Logo') %>
       </div>
       <div class="grid-col-auto font-sans-lg margin-right-305 display-inline-block">
-        <span>Submit your practice application by May 1, 2020!</span>
+        <span>Submit your practice application by June 15, 2020!</span>
       </div>
       <div class="grid-col-auto display-inline-block">
-        <button class="usa-button height-7 width-1505">Apply Now</button>
+        <span>
+          <form action="https://www.surveymonkey.com/r/ST20application" method="get" target="_blank">
+            <button style="width: 108px" class="usa-button height-7 width-1505" type="submit">Apply Now</button>
```

Comment:
we have a `style` attribute with a set `width` as well as a `class` with `width-1505`. what does `width-1505` do? if there is a width property in it, then it is being overwritten by the `style` attribute.

Reply:
this may be the mystery to josh's comment about the Apply Now being on two lines

Reply:
So when I updated the text to apply from May 1 to June 15... it forced the button to wrap to the next line... so I assumed that we wanted to keep it just right of the text if possible...  but yeah - by changing the width of the button...  the text within the button wrapped... if I went to width 109px the button wraps to the next line.

Reply:
So the mockup has a padding of `16px` for the button. Doing that, getting rid of the `width: 108px`, and adding `vertical-align:  middle` to the text span's div worked for me.

Reply:
<img width="678" alt="Screen Shot 2020-05-12 at 6 43 59 PM" src="https://user-images.githubusercontent.com/34111449/81752918-8ec19f00-9480-11ea-81dc-8046a8e7a1ba.png">

Reply:
Thanks Josh - it's still wrapping to the next line for me... but maybe I misinterpreted your instructions above...

<div class="grid-col-auto display-inline-block" style="vertical-align: middle">
        <span>
          <form action="https://www.surveymonkey.com/r/ST20application" method="get" target="_blank">
            <button style="padding: 16px" class="usa-button height-7 width-1505" type="submit">Apply Now</button>
          </form>
        </span>
      </div>

Reply:
Thanks again Josh!  styling for the button has been applied so that the button does not wrap to the next line - and the text in the button does not wrap.